README.md

```diff
@@ -1,2 +1,175 @@
+
+Before a PR can be merged, ensure it has been rebased to the latest version of the main branch. Once a PR has been merged, it should be closed and the feature branch deleted.
+
+#### Testing / Continuous integration
+Where relevant, tests should be written for all new pieces of functionality. (Use common sense to decide whether a test is necessary). A continuous integration service (e.g. [Codeship](https://codeship.com/), [Circle CI](https://circleci.com/)) will run these tests and deploy the code if all tests pass.
+
+A feature branch should only be merged if test coverage is comprehensive, and if all tests pass.
+
+### QA
+Once a developer considers a piece of work to be complete, it will have been merged into the main branch.
+
+At this point, the story should be reviewed on the development server once a branch has been merged, to ensure that the work is reviewed in an independent place (as opposed to on the laptop of the developer carrying out the work).
+
+In order to provide 'fresh eyes' on a feature, QA should be carried out by an internal member of the team but not by any developer that worked on the feature.
+
+The QA review should involve testing a piece of new functionality against the description and acceptance criteria outlined in the story. 
+
+If a story does not pass QA, a relevant note should be added so that the relevant developer knows what further work needs to be carried out.
+
+## Supporting services
+
+#### Trello
+Trello is used for managing stories into numerous boards. 
+
+The Trello boards should be kept up to date at all times as it is the best place to see an overview of the latest state of the website.
+
+#### Github
+Github is used for managing the codebase.
+
+Github PRs are used for reviewing and merging updates.
+
+Github issues are to be used only for very specific issues related to the current codebase. All major code updates should be managed via Trello tickets so that all development work can be tracked in one place.  
+
+#### Slack
+Slack is used for day-to-day communication between the team, rather than using email.
+
+Slack is transient in nature. As such it is easy to miss conversations. It should be used for day-to-day conversation, but any important updates should be communicated in meetings to ensure everyone is aware.
+
+#### Email
+Where possible email should be avoided for project-specific conversation. Where email is necessary, ensure everyone relevant is copied in. 
+
+Email should be used for admin, typically where permanent copies of documents are required by all involved parties - e.g. invoicing, agreeing terms of the project/contract, etc.
+
+## Any other business
+
+### Working remotely
+
+Before someone is able to work remotely, there must be agreement from all concerned. If someone is working remotely they are expected to keep to standard office hours, be present on Slack, and contactable via Skype.
```

Comment:
I find this part overly officious. Maybe something along the following lines would be more appropriate,

```
Depending on client preferences and other factors remote working is not always appropriate for a project, especially during early planning phases. But as long as the infrastructure is in place to support it (IM, video chats, GitHub etc.) we we think working remotely can be a viable and productive way to approach project work.
```